src/components/FlawAffects/FlawAffects.vue

```diff
@@ -0,0 +1,757 @@
+<script setup lang="ts">
+import { computed, toRefs, ref, watch } from 'vue';
+
+import { clone } from 'ramda';
+
+import FlawTrackers from '@/components/FlawTrackers.vue';
+import TrackerManager from '@/components/TrackerManager.vue';
+import LabelCollapsible from '@/components/widgets/LabelCollapsible.vue';
+import Modal from '@/components/widgets/Modal.vue';
+import FlawAffectsTable from '@/components/FlawAffects/FlawAffectsTable.vue';
+
+import { useModal } from '@/composables/useModal';
+import { usePaginationWithSettings } from '@/composables/usePaginationWithSettings';
+import { useFlawAffectsModel } from '@/composables/useFlawAffectsModel';
+
+import type { ZodAffectType, ZodFlawType } from '@/types';
+import { CVSS_V3 } from '@/constants';
+import { useSettingsStore } from '@/stores/SettingsStore';
+import { uniques, affectsMatcherFor } from '@/utils/helpers';
+
+import { displayModes } from './flawAffectConstants';
+import { useAffectSelections } from './useAffectSelections';
+import { useFilterSortAffects } from './useFilterSortAffects';
+
+const props = defineProps<{
+  embargoed: boolean;
+  error: null | Record<string, any>;
+  flaw: ZodFlawType;
+  relatedFlaws: ZodFlawType[];
+}>();
+
+const emit = defineEmits<{
+  'affect:add': [value: ZodAffectType];
+  'affect:recover': [value: ZodAffectType];
+  'affect:remove': [value: ZodAffectType];
+  'affect:track': [value: ZodAffectType];
+  'affects:refresh': [];
+}>();
+const settingsStore = useSettingsStore();
+const settings = ref(settingsStore.settings);
```

Comment:
`settingsStore.settings` is already a `ref`, since this is only read, I think is okay to destructure it
```suggestion
const { settigns } = useSettingsStore();
```

Reply:
You're right, that makes sense. I've made the change